user/serializers.py

```diff
@@ -22,3 +25,28 @@ def update(self, instance, validated_data):
             user.save()
 
         return user
+
+
+class AuthTokenSerializer(serializers.Serializer):
+    email = serializers.CharField(label=_("Email"), max_length=254)
+    password = serializers.CharField(
+        label=_("Password"), style={"input_type": "password"}
+    )
+
+    def validate(self, data):
+        email = data.get("email")
+        password = data.get("password")
+
+        if email and password:
+            user = authenticate(email=email, password=password)
```

Comment:
The `authenticate` function may not work as expected because it typically uses the 'username' field. Since your custom user model uses 'email' as the `USERNAME_FIELD`, ensure that your authentication backend supports email authentication or adjust the `authenticate` call accordingly.